### PR TITLE
Poprawa laboratorium Github

### PR DESCRIPTION
**Pull request by DanielSzymczak**
Repozytorjum było tworzone ponownie, z tego względu część działu 2. "Stałe i annotacje" była wykonana przeze mnie.

### CODE REVIEW

File: src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        return total;
+    }
+
+    protected void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+   protected boolean callCheck() {
+        return total == EMPTY;
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java:22]: Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).
This constant should be different (separate) than the one in peekaboo and countOut methods.

File: src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    public int getTotal() {
+        return total;
+    }
+
+    protected void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+   protected boolean callCheck() {
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java]: Visibility modifiers should be changed only for class fields. Classes without any visible methods and fields are useless.

File: src/main/java/edu/kis/vh/nursery/FIFORhymer.java [hunk 3]
```diff
@@ -1,21 +1,34 @@
...
+ * It implements a FIFO (First-In-First-Out) behavior for the rhymer.
+ */
+public class FIFORhymer extends DefaultCountingOutRhymer {
 
-public class FIFORhymer extends defaultCountingOutRhymer {
 
-    public defaultCountingOutRhymer temp = new defaultCountingOutRhymer();
-    
+    // Temporary rhymer used for implementing FIFO behavior
+    public DefaultCountingOutRhymer temp = new DefaultCountingOutRhymer();
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/FIFORhymer.java]: should be private and final

File: src/main/java/edu/kis/vh/nursery/HanoiRhymer.java [hunk 4]
```diff
@@ -1,17 +1,23 @@
...
-int totalRejected = 0;
+    private int totalRejected = 0;
 
     public int reportRejected() {
         return totalRejected;
     }
 
+    public int getTotalRejected() {
+        return totalRejected;
+    }
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/HanoiRhymer.java]: Task 10 in "accessory i hermetyzacja" concerned only 'total' getter.

File: src/main/java/edu/kis/vh/nursery/factory/RhymersFactory.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    
+        public DefaultCountingOutRhymer getStandardRhymer();
+        
+        public DefaultCountingOutRhymer getFalseRhymer();
+        
+        public DefaultCountingOutRhymer getFIFORhymer();
+        
+        public DefaultCountingOutRhymer getHanoiRhymer();
+        
+    }
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/factory/RhymersFactory.java]: Unnecessary indent.

**Review by kowallus — CHANGES_REQUESTED** [review]
Lack of changes in IntLinkedList class
Lack of encapsulation in Node class (Task 11 in "accessory i hermetyzacja")
More specific issues are inlined in the code.

Please fix issues by adding new commits.

**Comment by DanielSzymczak** [comment]
Wprowadziłem wymagane poprawki do gałęzi master.

**Pull request closed without merging**